src/frontend/apps/desk/src/components/Link.tsx

```diff
@@ -1,11 +1,16 @@
 import Link from 'next/link';
 import styled from 'styled-components';
 
+import { EnumTabIndex } from '@/types/components';
+
 export interface LinkProps {
   $css?: string;
+  tabIndex?: EnumTabIndex;
 }
 
-export const StyledLink = styled(Link)<LinkProps>`
+export const StyledLink = styled(Link).attrs<LinkProps>(({ tabIndex }) => ({
+  tabIndex: tabIndex ?? EnumTabIndex.keyboardNavigationOn,
+}))`
```

Comment:
Creating an enum for this check feels overly complex. I suggest improving the API of the LinkProps by using a boolean flag instead, for example:
```
export interface LinkProps {
  $css?: string;
  isKeyboardNavigationOn?: boolean;
}
```
Setting the default value to false ensures that the prop is straightforward to use. Exposing clear and concise props is crucial as it simplifies the component interface.

This approach makes it easier for developers to understand and utilize the component effectively.